packages/ng/icon/icon.component.html

```diff
@@ -0,0 +1,2 @@
+<span aria-hidden="true" class="lucca-icon icon-{{icon}} mod-{{size}} icon-color-{{color}}"></span>
+<span class="u-mask">{{alt}}</span>
```

Comment:
```suggestion
<span class="u-mask" *ngIf="alt">{{alt}}</span>
```

Reply:
I feel like this is pushing the dev into adding an alt but in the end, not using it if it's not here, meaning that in reality, it's optional. Idk, feels like lying on the API contract.

Reply:
The same logic could be applied to the alt attribute on images: it's mandatory, but can be empty. The idea behind this is to say explicitly: "this is a decorative image, but I'm saying it explicitly with an empty alt and not implicitly with a missing alt."

Reply:
(If you still think it's a nuance that can be overlooked, we can take the liberty of not making the parameter mandatory.)